packages/main/test/pages/Table_Acc.html

```diff
@@ -0,0 +1,211 @@
+			<ui5-table-cell><ui5-label>ChargeIt</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>Accessories</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #2b7c2b" role="status" aria-label="In Stock"><b>In Stock</b></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>29.99</b> EUR</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>0.15</b> kg</ui5-label></ui5-table-cell>
+			<ui5-table-row-action slot="actions" icon="edit" text="Edit" accessible-name="Edit product"></ui5-table-row-action>
+			<ui5-table-row-action-navigation slot="actions" interactive></ui5-table-row-action-navigation>
+		</ui5-table-row>
+
+		<ui5-table-row row-key="row2" navigated interactive>
+			<ui5-table-cell><ui5-label><b>USB-C Hub Adapter</b><br><a href="#" aria-label="Product ID HB-5012">HB-5012</a></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>ConnectPro</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>Connectivity</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #b3590a" role="status" aria-label="Low Stock"><b>Low Stock</b></ui5-table-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>34.50</b> EUR</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>0.08</b> kg</ui5-label></ui5-table-cell>
+			<ui5-table-row-action slot="actions" icon="edit" text="Edit" accessible-name="Edit product"></ui5-table-row-action>
+			<ui5-table-row-action-navigation slot="actions" interactive></ui5-table-row-action-navigation>
+		</ui5-table-row>
+
+		<ui5-table-row row-key="row3" interactive>
+			<ui5-table-cell><ui5-label><b>Bluetooth Speaker Mini</b><br><a href="#" aria-label="Product ID BS-2203">BS-2203</a></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>SoundWave</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>Audio</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #2b7c2b" role="status" aria-label="In Stock"><b>In Stock</b></ui5-table-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>59.99</b> EUR</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>0.25</b> kg</ui5-label></ui5-table-cell>
+		</ui5-table-row>
+
+		<ui5-table-row row-key="row4" interactive>
+			<ui5-table-cell><ui5-label><b>Wireless Mouse Pro</b><br><a href="#" aria-label="Product ID MT-2001">MT-2001</a></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label>TechGear Inc.</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-input value="Accessories" accessible-name-ref="categoryCol"></ui5-input></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label style="color: #2b7c2b" role="status" aria-label="In Stock"><b>In Stock</b></ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-label><b>45.99</b> EUR</ui5-label></ui5-table-cell>
+			<ui5-table-cell><ui5-slider min="0" max="10" show-tickmarks label-interval="2"></ui5-slider></ui5-table-cell>

```

Comment:
don't you get a complain from the ACC tools that this input has no labelling.  (we do not have a default labelling)